test/lit_tests/raising/affine_to_stablehlo5.mlir

```diff
@@ -0,0 +1,21 @@
+// RUN: enzymexlamlir-opt %s --raise-affine-to-stablehlo --arith-raise --enzyme-hlo-opt | FileCheck %s
```

Comment:
can we make the parallel bound sufficiently large that this doesn't constprop (and remains an iota)